### PR TITLE
New Command

### PR DESCRIPTION
**Pull request by salesawy1**
`?define` - Defines a word using Pearson's API

### CODE REVIEW

File: src/commands/Define.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    constructor(...args) {
+	    super(...args);
+
+	    this.aliases      = ['define'];
+	    this.module       = 'Fun';
+	    this.description  = 'Define a word.';
+	    this.usage        = 'define [word]';
+	    this.example      = 'define dyno';
+	    this.cooldown     = 3000;
+	    this.expectedArgs = 0;
```
Comment (briantanner) [inline src/commands/Define.js]: Expects 1 arg

File: src/commands/Define.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        let word = args[0];
+	    try {
+            let res = await superagent.get('http://api.pearson.com/v2/dictionaries/ldoce5/entries').query({'headword': word});
+            res = JSON.parse(res.text);
+            let definition;
+            let example;
+            let part_of_speech;
+
+            if (dynoWords.find(w => word.toLowerCase().startsWith(w))) {
+                //definition = this.dyno.globalConfig.definition || `Moderation. Music. Commands. Utilities. Fun. It's the best Discord bot™`;
```
Comment (briantanner) [inline src/commands/Define.js]: This should be uncommented

File: src/commands/Define.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                definition = `Moderation. Music. Commands. Utilities. Fun. It's the best Discord bot™`;
```
Comment (briantanner) [inline src/commands/Define.js]: and ditch this one

**Comment by salesawy1** [comment]
ok

**Comment by briantanner** [comment]
Amazing!

**Pull request merged**